### ISSUE TITLE
Help with properly including js please

### ISSUE THREAD

**Issue by daveisthemusic** [autoNumeric/autoNumeric#573]
Edit: Eventually found that I first need to look into installing Webpack on my server ([https://www.sitepoint.com/community/t/import-in-jquery-not-working/288000](url)). Readme a little confusing for rookies!

When using 
`<script src="https://unpkg.com/autonumeric"></script>` on my website, AutoNumeric works perfectly.

When wanting to include it from my own server, I have downloaded the master release, uploaded the whole folder via FTP, and used:
`<script src="https://myserver.com/autoNumeric-master/src/AutoNumeric.js"></script>`

after which, the .js file is included but I get the console error:

Unexpected identifier 'AutoNumericHelper'. import call expects exactly one argument.

(on line 49:) import AutoNumericHelper from './AutoNumericHelper';

and I can see that all the other .js files that get pulled in with the working version do not arrive here.

(I found on google to use `type="module"` with this import error, which just gives 

[Error] Failed to load resource: the server responded with a status of 503 (HTTP/2.0 503) (AutoNumericHelper, line 0) 
and
ReferenceError: Can't find variable: AutoNumeric)

Can someone tell me what mistake I'm making here please?

**Comment by AlexandreBonneau** [comment]
Indeed, you can either use the compiled library directly [via a CDN](https://github.com/autoNumeric/autoNumeric#in-the-browser), or include the source [in your own scripts](https://github.com/autoNumeric/autoNumeric#in-another-script), for latter compilation with your preferred bundler (Webpack, Rollup, Browserify, etc.).

I'm glad you solve that one on your own :)

**Comment by harrisyn** [comment]
@AlexandreBonneau I have been looking at this for a minute, and I must say I am still clueless as to how to use this in the browser, he may have solved this for himself, but your documentation says to just include the AutoNumeric.js file and it should work.... it doesn't

**Comment by AlexandreBonneau** [comment]
@harrisyn Have you checked the [specific chapter](https://github.com/autoNumeric/autoNumeric#in-the-browser) about this in the documentation?

If you include the `src/AutoNumeric.js` file, it won't work since the browsers still do not support the `import` keyword.
You need to include the `dist/autoNumeric.js` file (or the minified one) if you compile the library yourself, or just use any CDN as stated in the documentation.

Feel free to ask your question in the Gitter channel for quicker support.

**Comment by harrisyn** [comment]
Hi @AlexandreBonneau, I read that section, and I am most confused by it.
Allow me outline what I am looking to do, so you can assist me (and hopefully anyone else who stumbles on this).

- How do I include the "dist/autoNumeric.js" or compile it, since it isn't in the cloned repo.
- going past the issue above, how can I ensure that dynamically added fields with the same class name inherits the behavior from autoNumeric?

**Comment by AlexandreBonneau** [comment]
@harrisyn Since a few version, we stopped including the generated "dist" files in the git repository.
This is to make merging between branches easier (and also because it's a bad practice to include generated file into a source control).
When we push a commit, our CI automatically compile the dist file and those are published on various CDNs.

You can however generate the *dist* files by compiling those yourself following the [short explanation](https://github.com/autoNumeric/autoNumeric/#how-to-contribute) in the documentation:
To sum up:
```sh
git clone -b next https://github.com/autoNumeric/autoNumeric.git
cd autoNumeric
yarn install
yarn build # this is what builds the library
```

I'm not sure to follow your second question, could you please rephrase it?
If you want to generate more inputs with the same options, you can use the `.init()` function on the initial input.
There is no AutoNumeric watcher that automatically instantiate an AutoNumeric field when one with a specific class is created.

**Comment by gonebdg** [comment]
even with CDN it still show warning on console browser and the currency symbols doesn't work

```
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/autonumeric@4.5.4"></script>
<script type="text/javascript">
$(document).ready(function(){
	const autonumericoptions = {
		allowDecimalPadding: false,
		caretPositionOnFocus: 'start',
		currencySymbol: 'IDR',
		currencySymbolPlacement: AutoNumeric.options.currencySymbolPlacement.suffix,
		decimalCharacter: ',',
		digitGroupSeparator: '.',
		showWarnings: false
	};
	new AutoNumeric('#codvalue', autonumericoptions);
});

```
Console
```
DevTools failed to load SourceMap: Could not load content for https://cdn.jsdelivr.net/npm/autoNumeric.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
autonumeric@4.5.4:6 Warning: You are using the deprecated option name 'aSign'. Please use 'currencySymbol' instead from now on. The old option name will be dropped very soon™.
autonumeric@4.5.4:6 Warning: You are using the deprecated option name 'aDec'. Please use 'decimalCharacter' instead from now on. The old option name will be dropped very soon™
autonumeric@4.5.4:6 Warning: You are using the deprecated option name 'aSep'. Please use 'digitGroupSeparator' instead from now on. The old option name will be dropped very soon™.
```

**Comment by OberstVilla** [comment]
The CDN distributed version still does not properly expose some static methods (see previous errors).
If I download it from the Git repository, I get the AutoNumericHelper not found error.

**Comment by OberstVilla** [comment]
So the way I see it at the moment we have a really powerful library with hundreds of options but no way to install it so it functions properly.

**Comment by AlexandreBonneau** [comment]
Have you checked the [live example](https://codepen.io/AnotherLinuxUser/pen/JyBGpz?editors=1010) where using a CDN does not output any errors?

For info in this example I'm using the `https://unpkg.com/autonumeric` CDN.

**Comment by OberstVilla** [comment]
Yes, thank you. I have found that one and it works fine.
